libcalico-go/lib/apis/k8sconvert/k8snetworkpolicy.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sconvert
+
+import (
+	"github.com/projectcalico/calico/libcalico-go/lib/apis/v1/unversioned"
+
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var VersionK8sNetworkingV1 = "networking.k8s.io/v1"
+
+type K8sNetworkPolicy struct {
+	unversioned.TypeMetadata
+	Metadata K8sNetworkPolicyMetadata       `json:"metadata,omitempty"`
+	Spec     networkingv1.NetworkPolicySpec `json:"spec,omitempty"`
+}
+
+func NewK8sNetworkPolicy() *K8sNetworkPolicy {
```

Comment:
I think I'd prefer to keep this localized to calicoctl for now - no need for it in libcalico AFAICT - we can just put this right in the policy coversion package.

Reply:
Got it, thanks @caseydavenport! Makes sense, will do!